arcgis-ios-sdk-samples/Edit data/Edit features with feature-linked annotation/README.md

```diff
@@ -0,0 +1,46 @@
+# Edit features with feature-linked annotation
+
+Edit feature attributes which are linked to annotation through an expression.
+
+![Image of edit features with feature-linked annotation](feature-linked-annotation.png)
+
+## Use case
+
+Annotation is useful for displaying text that you don't want to move or resize when the map is panned or zoomed (unlike labels which will move and resize). Feature-linked annotation will update when a feature attribute referenced by the annotation expression is also updated. Additionally, the position of the annotation will transform to match any transformation to the linked feature's geometry.
+
+## How to use the sample
+
+Pan and zoom the map to see that the text on the map is annotation, not labels. Tap one of the address points to update the house number (AD\_ADDRESS) and street name (ST\_STR\_NAM). Tap one of the dashed parcel polylines and tap another location to change its geometry. NOTE: Selection is only enabled for points and straight (single segment) polylines. The feature-linked annotation will update accordingly.
```

Comment:
I'm a bit confused about the field names. In code they are `AD_ADDRESS` and `ST_STR_NAM`, but here they have backslash. Should they be unified?

Reply:
The backslashes allow them to be rendered correctly. Without them, half of the phrases were italicized.

Reply:
Ha nice catch! I see. Thanks. 😄